components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -579,7 +579,7 @@ def leave_none_unset_int(s):
           " 'webtest/webclient_plugins/center_plugin.overlay.js.html',"
           " 'channel_overlay_panel']``. "
           "The javascript loads data into ``$('#div_id')``.")],
-    "omero.web.viewer.initial_zoom_level":
+    "omero.client.viewer.initial_zoom_level":
```

Comment:
Does it help to upgrade etc/grid/config.xml as I've done in https://github.com/openmicroscopy/openmicroscopy/pull/3602 ?